wagtail/admin/views/collections.py

```diff
@@ -182,6 +182,6 @@ def post(self, request, pk):
             # collection is non-empty; refuse to delete it
             return HttpResponseForbidden()
 
+        messages.success(request, self.get_success_message())
```

Comment:
note:
`get_success_message` is defined on the `DeleteView` and does the necessary interpolation
https://github.com/wagtail/wagtail/blob/main/wagtail/admin/views/generic/models.py#L693-L696

Reply:
Yeah I think this code probably did not use the shared views historically so it has a mix of code that uses it and ... a mix of codes that does not :)